OTAnalytics/plugin_ui/customtkinter_gui/dummy_viewmodel.py

```diff
@@ -435,19 +423,16 @@ def edit_flow(self) -> None:
             position = position = self._treeview_flows.get_position()
             InfoBox(message="Please select a flow to edit", initial_position=position)
             return
-        flow = parse_flow_id(selected_flow)
-        if from_section := self._application.get_section_for(
-            SectionId(flow.from_section)
-        ):
-            self._edit_flow(flow, from_section)
+        flow_id = FlowId(selected_flow)
+        if flow := self._application.get_flow_for(flow_id):
+            self._edit_flow(flow)
 
-    def _edit_flow(self, flow: FlowId, from_section: Section) -> None:
-        distances = from_section.plugin_data.get(DISTANCES, {})
-        distance: str = distances.get(flow.to_section, {})
+    def _edit_flow(self, flow: Flow) -> None:
         input_data = {
-            START_SECTION: flow.from_section,
-            END_SECTION: flow.to_section,
-            DISTANCE: distance,
+            FLOW_ID: flow.id.id,
+            START_SECTION: flow.start.id.id,
```

Comment:
I find `id.id` not really self explaining. The second `id` gives me the first `id` as a string, right?

Reply:
Switched to `serialize` method.